packages/next/build/webpack-config.ts

```diff
@@ -980,6 +980,12 @@ export default async function getBaseWebpackConfig(
     },
   }
 
+  const rscCodeCondition = {
+    test: codeCondition.test,
```

Comment:
This will always be overridden by other `test` conditions. We can just:  
```suggestion
    test: serverComponentsRegex,
```